src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java

```diff
@@ -191,4 +175,9 @@ public URL getRemoteHost() {
         return null;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) This class overrides "equals()" and should therefore also override "hashCode()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1206)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1185)